src/modules/dashboard/components/CoinMachine/TokenSalesTable/TokenSalesTable.tsx

```diff
@@ -169,7 +168,6 @@ const TokenSalesTable = ({
   useEffect(() => {
     if (periodRemainingMS > 1000 && periodRemainingMS < periodLengthMS) {
       setTimeout(() => {
-        refetchSalePeriodsData(salePeriodQueryVariables);
         startPollingSalePeriodsData(periodLengthMS);
       }, periodRemainingMS);
     } else {
```

Comment:
If we don't need the refetch anymore, I think this code can be further simplified like this:

```
  useEffect(() => {
    startPollingSalePeriodsData(periodLengthMS);
    return () => stopPollingSalePeriodsData();
  }, [
    periodLengthMS,
    startPollingSalePeriodsData,
    stopPollingSalePeriodsData,
  ]);
```

As far as I understand this code, the `else` would still be doing the same thing as the first conditional